app/src/main/java/it/niedermann/owncloud/notes/edit/NoteEditFragment.java

```diff
@@ -166,11 +163,28 @@ protected void onNoteLoaded(Note note) {
         }
     }
 
+    private void openSoftKeyboard() {
+        binding.editContent.postDelayed(() -> {
+            binding.editContent.requestFocus();
+
+            final InputMethodManager imm = (InputMethodManager) requireContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+            if (imm != null) {
+                imm.showSoftInput(binding.editContent, InputMethodManager.SHOW_IMPLICIT);
+            } else {
+                Log.e(TAG, InputMethodManager.class.getSimpleName() + " is null.");
+            }
+            //Without a small delay the keyboard does not show reliably
+        }, 100);
```

Comment:
In which case does it not work reliable? I didn't experience an issue prior to this PR, and i'd like to avoid a delay - even 100ms will make feeling this just laggy...

Reply:
The problem seems to be device dependent.
On my phone the keyboard would not appear when going to background then foreground and not always when creating a new note.
On an emulator it worked fine with both.

I think it's a race conditions problem but I was not able to find the root cause of this.

Reply:
Yep, i was able to confirm those issues. Let's go with `100ms` until we have a better solution :slightly_smiling_face: